addons/web/static/src/js/fields/basic_fields.js

```diff
@@ -2022,7 +2022,8 @@ var JournalDashboardGraph = AbstractField.extend({
                         .y(function (d) { return d.value; })
                         .showValues(false)
                         .showYAxis(false)
-                        .margin({'left': 0, 'right': 0, 'top': 0, 'bottom': 40});
+                        .color(['#875A7B', '#526774', '#FA8072'])
```

Comment:
My first idea was: inject fake elements in the DOM, extract the values using `$.css` and the remove them back. It's ugly though...  I preferred to leave it as it is for the moment.
At least adding a not should be necessary, you're right